$NetBSD$

  - Avoid warning about 'assignment discards qualifiers from pointer target
    type'. The SDL function SDLNet_ResolveIP() returns a 'const' value.

--- src/SDL.xs.orig	2005-10-05 11:25:49.000000000 +0200
+++ src/SDL.xs
@@ -2441,7 +2441,7 @@ NetFreeIPaddress ( ip )
 	CODE:
 		safefree(ip);
 
-char*
+const char*
 NetResolveIP ( address )
 	IPaddress *address
 	CODE:
